### PR TITLE
WIP: [BAC-580] Added control plane

### PR DESCRIPTION
**Pull request by cohandv**

### CODE REVIEW

File: charts/base/templates/daemonset.yaml [hunk 1]
```diff
@@ -95,8 +96,37 @@ spec:
...
+              valueFrom:
+                fieldRef:
+                  apiVersion: v1
+                  fieldPath: spec.nodeName
+            - name: NULLPLATFORM_APIKEY
+              valueFrom:
+                secretKeyRef:
+                  key: NULLPLATFORM_APIKEY
+                  name: nullplatform-secret
+            # Not sure if this is required, I'd leave this out for now
```
Comment (cohandv) [inline charts/base/templates/daemonset.yaml:114]: @geisbruch validate if this is needed in the context of Azure at least

File: charts/base/values.yaml [hunk 2]
```diff
@@ -26,8 +26,13 @@ gateway:
     autoscaling:
       minReplicas: 2
       maxReplicas: 10
+# Nullplatform configurations
+nullplatform:
+  apiKey: ""
```
Comment (cohandv) [inline charts/base/values.yaml:31]: we need to include this on infrastructure-configurations repo

File: charts/base/values.yaml [hunk 3]
```diff
@@ -26,8 +26,13 @@ gateway:
...
 # Logging configurations
 logging:
+  agent:
```
Comment (geisbruch) [inline charts/base/values.yaml:34]: no lo pondria dentro de login, creo que deberia tener su categoria de config

File: charts/base/templates/daemonset.yaml [hunk 4]
```diff
@@ -95,8 +96,37 @@ spec:
                   key: LOKI_BEARER_TOKEN
             {{- end}}
             {{- end}}
-        - name: nullplatform-logging-metrics-exporter
-          image: {{ .Values.logging.metricsExporter.image }}
+        - name: nullplatform-controlplane-agent
```
Comment (geisbruch) [inline charts/base/templates/daemonset.yaml:99]: Si no tenes apikey configurada o alguna otra property no deberia levantarlo

File: charts/base/templates/daemonset.yaml (hunk 1 above)
Comment (geisbruch) [reply charts/base/templates/daemonset.yaml:114]: Not required for azure and also I think we can omit it for aws, we should do double check why is in use I think It's for self discovery, in K8S may we should use an SA

**Pull request merged**